src/test_workflow/benchmark_test/benchmark_args.py

```diff
@@ -69,6 +71,8 @@ def __init__(self) -> None:
                             help="A comma-separated list of key=value pairs that will be added to jvm.options as JVM system properties.")
         parser.add_argument("--additional-config", nargs='*', action=JsonArgs, dest="additional_config",
                             help="Additional opensearch.yml config parameters passed as JSON")
+        parser.add_argument("--use-50-percent-heap", dest="use_50_percent_heap", action="store_true",
```

Comment:
Can we just use `fifty` instead of `50` to uniform style?

Reply:
And fifty percent heap should be default all the time if not specify?

Reply:
I think its better to match cluster-cdk params to avoid confusion https://github.com/opensearch-project/opensearch-cluster-cdk/pull/23

Reply:
Not aware this is part of the cluster cdk param.